generators/server/templates/src/main/java/package/repository/UserRepository.java.ejs

```diff
@@ -214,33 +240,72 @@ public interface UserRepository extends <% if (databaseType === 'sql') { %>JpaRe
     <% if (!reactive) { %>// See https://github.com/neo4j/sdn-rx/issues/51<% } %>
     <%= listOrFlux %><<%= asEntity('User') %>> findAll();
 
+    <%_ } _%>
+    <%_ if (databaseType === 'couchbase') { _%>
+    @Override
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+    <%= listOrFlux %><<%= asEntity('User') %>> findAll();
+
+    <%_ } _%>
+    <%_ if (databaseType === 'couchbase') { _%>
+    @Override
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+    <%= listOrFlux %><<%= asEntity('User') %>> findAll(Sort sort);
+
     <%_ } _%>
     <%_ if (databaseType === 'sql') { _%>
     @EntityGraph(attributePaths = "authorities")
         <%_ if (cacheManagerIsAvailable === true) { _%>
     @Cacheable(cacheNames = USERS_BY_LOGIN_CACHE)
         <%_ } _%>
+        <%_ if (databaseType === 'couchbase') { _%>
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+        <%_ } _%>
     Optional<<%= asEntity('User') %>> findOneWithAuthoritiesByLogin(String login);
 
         <%_ if (authenticationType !== 'oauth2') { _%>
     @EntityGraph(attributePaths = "authorities")
             <%_ if (cacheManagerIsAvailable === true) { _%>
     @Cacheable(cacheNames = USERS_BY_EMAIL_CACHE)
             <%_ } _%>
+        <%_ if (databaseType === 'couchbase') { _%>
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+        <%_ } _%>
     Optional<<%= asEntity('User') %>> findOneWithAuthoritiesByEmailIgnoreCase(String email);
 
         <%_ } _%>
     <%_ } _%>
 
     <% if (reactive) { %>
+        <%_ if (databaseType === 'couchbase') { _%>
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+        <%_ } _%>
     Flux<<%= asEntity('User') %>>findAllByIdNotNull(Pageable pageable);
 
+        <%_ if (databaseType === 'couchbase') { _%>
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+    @Query("#{#n1ql.selectEntity} WHERE #{#n1ql.filter} AND meta(#{#n1ql.bucket}).id is not null AND activated = true")
+        <%_ } _%>
     Flux<<%= asEntity('User') %>>findAllByIdNotNullAndActivatedIsTrue(Pageable pageable);
 
     Mono<Long> count();
     <% } else { %>
+        <%_ if (databaseType === 'couchbase') { _%>
+    @ScanConsistency(query = QueryScanConsistency.REQUEST_PLUS)
+    @Query("#{#n1ql.selectEntity} WHERE #{#n1ql.filter} AND meta(#{#n1ql.bucket}).id is not null AND activated = true")
```

Comment:
I've removed all instanced of the `meta().id` check from there and changed the naming convention for those and related ones across the `UserRepository` and `UserService` files.

Reply:
Unresolved this thread since renaming the query to `findAllByActivatedIsTrue` seems to have caused some issues with the other databases as the SQL tests are failing on my fork with an error along the lines of:
```
No property findAllActivated found for type User!
```
Currently attempting to fix it.

Reply:
I've fixed it by only switching to `findAllByActivatedIsTrue` for Couchbase but leaving it as `findAllByIdNotNullAndActivatedIsTrue` for the other databases